code/style.css

```diff
@@ -1,7 +1,7 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: #6ECB63;
+  --secondary: #B1E693;
```

Comment:
from what I can see you haven't changed that much in the css, which is fine but you could probably make the page more responsive when the game is on a mobile